subprojects/docs/src/docs/userguide/running-builds/gradle_daemon.adoc

```diff
@@ -197,35 +197,35 @@ A significant part of the story for modern JVM performance is runtime code optim
 
 The Daemon also allows more effective in memory caching across builds. For example, the classes needed by the build (e.g. plugins, build scripts) can be held in memory between builds. Similarly, Gradle can maintain in-memory caches of build data such as the hashes of task inputs and outputs, used for incremental building.
 
-When <<sec:daemon_watch_fs,watching the file system>> is enabled, the Daemon can re-use information about the file system from the last build.
-For incremental builds, there are only a few changes between builds.
-Therefore, incremental builds can re-use most of the information about the file system from the last build and benefit the most from watching the file system.
+To detect changes on the file-system, and to calculate what needs to be rebuilt, Gradle collects a lot of information about the state of the file-system during every build.
```

Comment:
I don't like repeating "the file-system" here, but have no good idea how to remove the redundancy.